packages/navigation/src/NavigationMenuSelect.tsx

```diff
@@ -0,0 +1,52 @@
+import type { CollectionBase } from '@react-types/shared';
```

Comment:
I had done a `git mv` on this file (from `NavigationMenuDropdown` to `NavigationMenuSelect`), but it was lost along the way (in retrospect I probably should have done the rename as a separate commit)

...if we're worried about it I can rebase these changes to make sure the move is reflected, but I _think_ our default merge strategy will squash the commits which will likely result in the old file being deleted and this new one being added.